### PR TITLE
Update animals table

### PR DESCRIPTION
**Pull request by YKerroum**
- [ ] Add column 'species'.
- [ ] Insert the new animals to the table.
- [ ] Add transaction to update the animals table by setting the species column to unspecified:
Q1
- [ ] Rollback:
Q2

- [ ] Update the animals table by setting the species column to digimon or pokemon and commit:
Q3

- [ ] Add a transaction to delete all records in the animals table then rollback:
Q4

- [ ]  Add a transaction to delete all animals born after Jan 1st, 2022, create savepoint, update weight, rollback , update weight then commit: 
Q5

- [ ] How many animals are there?
Query1

- [ ] How many animals have never tried to escape?
Query2

- [ ] What is the average weight of animals?
Query3

- [ ] Who escapes the most, neutered or not neutered animals?
Query4

- [ ] What is the minimum and maximum weight of each type of animal?
Query5

- [ ] What is the average number of escape attempts per animal type of those born between 1990 and 2000?
Query6

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -5,3 +5,12 @@ VALUES ('Agumon','2020-02-03',0,true, 10.23),
 ('Gabumon','2018-11-15',2,true, 8.0),
 ('Pikachu','2021-01-07',1,false,15.04),
 ('Devimon','2017-05-12',5,true,11.0);
+
+INSERT INTO animals (name,date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Charmander','2020-02-08',0,false, 11),
+('Plantmon','2021-11-15',2,true, 5.7),
+('Squirtle','1993-04-02',3,false,12.13),
+('Angemon','2005-06-12',1,true,45),
```
Comment (BenMukebo) [inline data.sql:13]: - [ ] Great job so far however you need to pair more attention to the project requirements. The fourth first data should have a negative weight_kg value and not a positive value. Kindly correct this 👍🏻Your code might look like this:

```sql
    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Charmander', '2020/2/8', '0', 'false', '-11.0');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Plantmon', '2021/11/15', '2', 'true', '-5.7');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Squirtle', '1993/04/02', '3', 'false', '-12.13');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Angemon', '2005/06/12', '1', 'true', '-45.0');
```

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @YKerroum,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
- Professional README ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (YKerroum) [reply data.sql:13]: thank you done.

**Review by ShadyShawkat — APPROVED** [review]
## Hi @YKerroum, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**